lib/internal/process.js

```diff
@@ -60,7 +60,7 @@ function setupConfig(_source) {
       .replace(/"/g, '\\"')
       .replace(/'/g, '"');
 
-  process.config = JSON.parse(config, function(key, value) {
+  process.config = exports.config = JSON.parse(config, (key, value) => {
```

Comment:
If we want to be robust against users clobbering the runtime environment, then we shouldn't take half measures; either we guard against it or we don't.